adminguide.rst

```diff
@@ -312,136 +356,104 @@ This space should be preserved across a reboot as it contains the Flux
 
  $ flux account add-bank root 1
  $ flux account add-bank --parent-bank=root sub_bank_A 1
 
-We can then add some users to those banks so those users can run jobs on the
-system:
+Users that are permitted to run on the system must be assigned banks,
+for example:
 
 .. code-block:: console
 
  $ flux account add-user --username=user1234 --bank=sub_bank_A
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Loading the multi-factor priority plugin
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The other component to load is the multi-factor priority plugin, which can be
-loaded by specifying it in the ``job-manager.plugins`` section of the Flux TOML
-configuration file. Below is an example configuration file:
+Multi-factor priority plugin
+----------------------------
 
-``/etc/flux/system/conf.d/plugins.toml``
+When flux-accounting is installed, the job manager uses a multi-factor
+priority plugin to calculate job priorities.  The plugin must be listed in
+the job manager config file:
 
 .. code-block:: toml
 
+ # /etc/flux/system/conf.d/job-manager.toml
+
  [job-manager]
  plugins = [
-   { load = "/usr/lib64/flux/job-manager/plugins/mf_priority.so" },
+   { load = "mf_priority.so" },
  ]
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Setting up the automatic update scripts
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-There are three update commands and scripts that require setup in order for the
-database to be periodically updated with new job usage and fairshare
-information. These commands and scripts should be run as the flux user and they
-should be run in order and with the same frequency to ensure accurate updates.
-A suggested frequency for running these automatic updates would be to run these
-once every 30 minutes, but can be tuned to run more or less frequently.
+Automatic updates
+-----------------
 
-The first automatic cron job that should run is the ``update-usage`` subcommand.
-This subcommand fetches the most recent job records from the job-archive
-database for every user in the flux-accounting database and calculates a new job
-usage value. This subcommand takes one optional argument,
-``--priority-decay-half-life`` which should be set to the same value as the
-``--priority-decay-half-life`` parameter mentioned above when first creating the
-database; it represents the number of weeks to hold one job usage "chunk."
-If not specified, this optional argument also defaults to 1 week.
+A series of actions should run periodically to keep the accounting
+system in sync with Flux:
 
-.. code-block:: console
+- The job-archive module scans inactive jobs and dumps them to a sqlite
+  database.
+- A script reads the archive database and updates the job usage data in the
+  accounting database.
+- A script updates the per-user fair share factors in the accounting database.
+- A script pushes updated factors to the multi-factor priority plugin.
 
- $ flux account update-usage
+The ``job-archive`` module must be configured to run periodically:
```

Comment:
I like the inclusion of the `job-archive` module here, thanks for including this. I think this drives home the use for the `update-usage` command and how it works. Perhaps in the near future, flux-accounting can have more detailed descriptions on a separate page of how it calculates job priorities using the archive DB?